.travis.yml

```diff
@@ -1,13 +1,62 @@
 language: php
 
+services:
+  - mysql
+
 php:
-  - 5.4
-  - 5.5
   - 5.6
-  - hhvm
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
+  - 7.4
+
+env:
+  - LARAVEL_VERSION=5.6.*
```

Comment:
We also have to have Laravel 5.4 and 5.5.

Reply:
I will add L5.4 and L5.5 as I progress them, getting the matrix out in a correct way is not so easy

Reply:
Okay!